src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/RememberMeFactory.php

```diff
@@ -104,7 +105,7 @@ public function create(ContainerBuilder $container, $id, $config, $userProvider,
             throw new \RuntimeException('You must configure at least one remember-me aware listener (such as form-login) for each firewall that has remember-me enabled.');
         }
 
-        $rememberMeServices->replaceArgument(0, array_unique($userProviders));
+        $rememberMeServices->replaceArgument(0, new IteratorArgument(array_unique($userProviders)));
```

Comment:
doesn't this require a bump of the minimum version of security-http?

Reply:
right! release happened meanwhile, fixed